### ISSUE TITLE
QA PH3_Accessibility: Attending - Unselected Event/Travel tabs not accessible with keyboard navigation 

### ISSUE THREAD

**Issue by brittasinnott** [GoogleChrome/ioweb2016#631]
On the Attending pages, the unselected tab (either Event or Travel) is not accessible using the keyboard only to navigate. For example, if user is landed on the Event page, the Travel page is inaccessible. 

Steps: 
1. Go to https://io-webapp-staging.appspot.com/io2016/
2. Select Attending 
3. Use the keyboard only to navigate 
4. Notice that unselected tab (Event or Travel) is skipped

Chrome 49 Mac OS X  10.11 

![gio_631](https://cloud.githubusercontent.com/assets/15128124/14443411/eb548da2-fff3-11e5-87ce-f4e243bab487.png)
